app/controllers/site_controller.rb

```diff
@@ -1,62 +1,82 @@
 class SiteController < ApplicationController
+  respond_to? :json, :html
   before_action :load_habitat
   before_action :load_global
   before_action :load_charts_data
 
-  def warmwater
-    @data = YAML.load(File.open("#{Rails.root}/lib/data/content/warmwater.yml", 'r'))
+  def index
+    @title = @habitat.title
 
-    @commitments = [
-      @aichi_targets,
-      @sdgs,
-      @data['other_targets']
-    ]
-  end
-
-  def saltmarshes
-    @data = YAML.load(File.open("#{Rails.root}/lib/data/content/saltmarshes.yml", 'r'))
-
-    @commitments = [
-      @aichi_targets,
-      @sdgs,
-      @data['other_targets']
-    ]
-  end
-
-  def mangroves
-    @data = YAML.load(File.open("#{Rails.root}/lib/data/content/mangroves.yml", 'r'))
-
-    @commitments = [
-      @aichi_targets,
-      @sdgs,
-      @data['other_targets']
-    ]
-  end
-
-  def seagrasses
-    @data = YAML.load(File.open("#{Rails.root}/lib/data/content/seagrasses.yml", 'r'))
-
-    @commitments = [
-      @aichi_targets,
-      @sdgs,
-      @data['other_targets']
-    ]
-  end
-
-  def coldcorals
-    @data = YAML.load(File.open("#{Rails.root}/lib/data/content/coldwater.yml", 'r'))
-
-    @commitments = [
-      @aichi_targets,
-      @sdgs,
-      @data['other_targets']
-    ]
+    @nav = [
```

Comment:
Since this is static data, I think we could put this into a constant or a YML file.